### ISSUE TITLE
Package not compatible with Vue cli version 5.x

### ISSUE THREAD

**Issue by dvago** [FranckFreiburger/vue-pdf#363]
The newer version of vue-cli 5.x migrates from webpack 4 to webpack 5, the newer version of webpack does change the support for web workers.

So, the latest version of vue-pdf throws the following error:
```sh
ERROR in ./node_modules/pdfjs-dist/es5/build/pdf.worker.js (./node_modules/vue-pdf/node_modules/worker-loader/dist/cjs.js!./node_modules/pdfjs-dist/es5/build/pdf.worker.js)

Module build failed (from ./node_modules/vue-pdf/node_modules/worker-loader/dist/cjs.js):
Error: MainTemplate.hooks.hotBootstrap has been removed (use your own RuntimeModule instead)
```

Would you please update this package to make it run with vue-cli 5?

**Comment by liugj101** [comment]
same problem as you, do I have to downgrade to VUE-CLI4.0

**Comment by YSoli** [comment]
Same issue here while migrating to vue-cli 5.x

**Comment by kstraszewski** [comment]
Same here

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/10195995/161639475-9c5ae7ea-1198-45ca-8056-c8e91a284256.png">

**Comment by xuxusheng** [comment]
same +1

**Comment by EugeneKuzmin** [comment]
+1

**Comment by Glandos** [comment]
This is strange. I don't know what I did, but it works for me.
However, it uses worker-loader which outputs `DEP_WEBPACK_COMPILATION_CACHE` and `DEP_WEBPACK_DEPRECATION_ARRAY_TO_SET_INDEXER` with Webpack 5 brought by vue-cli 5.

**Comment by Byteschmiede** [comment]
same +1

**Comment by dvago** [comment]
@liugj101 I ended up forking the project, cleaning up the code and make it work for `vue-cli` 5, this `vue-pdf` project is clearly dead, it's pretty pointless waiting for any maintainer doing anything.

**Comment by cwilby** [comment]
@dvago Are you able to share those changes?  Not seeing changes here: https://github.com/dvago/vue-pdf

**Comment by dvago** [comment]
Not really, I created a package within my company git account and stripped out lots of features that we don't use to run it as simple as possible, so the package is not called `vue-pdf` anymore and it's not public. 

I don't think it makes sense for me to publish a clone of it as I'm not going to be able to maintain it or keep up with the eventual broad public requests to be fair, hope that makes sense.

**Comment by MoNaiZi** [comment]
+1

**Comment by chenweigh** [comment]
+1 
ERROR in ./node_modules/pdfjs-dist/es5/build/pdf.worker.js (./node_modules/vue-pdf/node_modules/worker-loader/dist/cjs.js!./node_modules/pdfjs-dist/es5/build/pdf.worker.js)

**Comment by brizer** [comment]
Are there any other good solutions besides using vue-pdf-embed?

**Comment by ziyang314** [comment]
+1

**Comment by shubhammatta** [comment]
+1
